File: startingPoint/01-EstruturasCondicionais/exercicios_propostos/01.py
# 1. Faça um programa que receba três notas de um aluno, calcule e mostre a média aritmética e a mensagem constante na tabela a seguir. Aos alunos que ficaram para exame, calcule e mostre a nota que deverão tirar para serem aprovados, considerando que a média exibida é 6,0.

#MÉDIA ARITMÉTICA	MENSAGEM
#0,0 +-----------+ 3,0	Reprovado
#3,0 +----------+ 7,0	Exame
#7,0 +-----------\* 10,0	Aprovado


nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))
nota3 = float(input("Digite a terceira nota: "))

media = (nota1 + nota2 + nota3) / 3
media = round(media, 2)

print("A média aritimética é: ", media)

if media >= 0 and media < 3:
    print("reprovado.")
if media >= 3 and media < 7:
    print("Exame.")
    nota_exame = 12 - media
    print("Deve tirar nota", nota_exame, "para ser aprovado")
if media >=7 and media <= 10:
    print("Aprovado!")

File: SRC - PERL/00 - Estruturas Sequenciais/01.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);


# Faça um programa que receba três notas, calcule e mostre a média aritimética entre elas

print("Digita a 1 nota > ");
my $nota1 = <>;
chomp $nota1;

print("Digita a 2 nota > ");
my $nota2 = <>;
chomp $nota2;

print("Digita a 3 nota > ");
my $nota3 = <>;
chomp $nota3;

my $media = ($nota1 + $nota2 + $nota3) / 3.0;

say "A média das notas é ", $media;
